### ISSUE TITLE
"nsfileprovidererrordomain -1000" and "you need to authenticate before accesing"

### ISSUE THREAD

**Issue by pedrozandonai** [nextcloud/ios#2685]
### Steps to reproduce

1. Just open the finder on the iPhone and tap on the Nextcloud local folder

I'm trying to access my files on nextcloud iOS folder but the error "nsfileprovidererrordomain -1000" keeps showing up, sometimes also appears "you need to authenticate before accesing".

### Environment data

**iOS version:** 17.1.1

**Nextcloud iOS app version:** 4.9.2.20

**Server operating system:** dietpi

**Web server:** lighttpd

**Database:** mariadb

**PHP version:** 8.2.7

**Nextcloud version:** 27.1.4

**Comment by nepumax** [comment]
Yeah, got this today. Just on the iPad, not on the iPhone. Disconnected and reconnect all devices. No change at all.
